static/js/services/videowaiter.js

```diff
@@ -46,13 +47,17 @@ define(["underscore"], function(_) {
 				if (videoTracks.length > 0 && this.count >= 10) {
 					var videoTrack = videoTracks[0];
 					if (videoTrack.enabled === true && videoTrack.muted === true) {
+						videoTrack.onunmute = function() {
+							videoTrack.onunmute = undefined;
+							recheck();
```

Comment:
What's the problem with running `cb` a second time? Every consumer of the video waiter seems to have no problem with that. We could add a note, so the behaviour should be clear and documented. Maybe a second parameter for the `cb` function can be added, so the consumers at least have a way to handle the retrigger differently.